apps/auth/src/guards/jwt-auth.guard.ts

```diff
@@ -32,11 +36,18 @@ export class JwtAuthGuard extends AuthGuard('jwt') {
         } catch (error) {
             if (error instanceof TokenExpiredError) {
                 const { userId } = jwtService.decode(token);
-                const user = await this.authService.regenerateToken(userId, response);
+                const user = this.usersService.getUserById({ id: userId });
+                const newToken = this.tokenService.createToken({ userId });
+                const expires = new Date();
```

Comment:
Let's move logic for creating expires to the token service as well, and then we can return { token, expires } from createToken method